### PR TITLE
Add constraint to monolog-bundle

### PR DESCRIPTION
**Pull request by notFloran**
See #33

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -16,20 +16,28 @@
...
         "symfony/config":"^3.0 || ^4.0",
         "symfony/http-foundation": "^3.0 || ^4.0",
         "symfony/dependency-injection": "^3.0 || ^4.0",
-        "symfony/http-kernel": "^3.0 || ^4.0"
+        "symfony/http-kernel": "^3.0 || ^4.0",
+        "symfony/monolog-bundle": "^2.8 || ^3.0"
     },
     "require-dev": {
         "phpspec/phpspec": "^2.5|^3.0",
         "phpstan/phpstan": "^0.8.0"
```
Comment (nclsHart) [inline composer.json:29]: This comment is not directly related to this PR but why do we have this requirement in dev?

**Review by nclsHart — APPROVED** [review]

File: composer.json (hunk 1 above)
Comment (notFloran) [reply composer.json:29]: I add it with #23, I think we can now remove it
